tenderly-suite/token-transfers/token-transfers/handleTokenTransfer.js

```diff
@@ -0,0 +1,156 @@
+const { ethers } = require('ethers');
+const axios = require('axios').default;
+
+const ERC20_ABI = ['function symbol() external view returns (string)', 'function decimals() external view returns (uint8)'];
+const ITS_ABI = ['function getTokenAddress(bytes32 tokenId) external view returns (address)'];
+
+const COIN_MARKET_QUOTES_URL = 'https://pro-api.coinmarketcap.com/v2/cryptocurrency/quotes/latest';
+const PAGER_DUTY_ALERT_URL = 'https://events.pagerduty.com/v2/enqueue';
+
+const TOKEN_THRESHOLD_WITH_PRICE = [50000, 100000, 500000]; // [info, warning, critical]
+const TOKEN_THRESHOLD_WITHOUT_PRICE = [500000, 1000000, 5000000]; // [info, warning, critical]
+
+const TOPIC_0_TOKEN_SENT = '0x9df6ec8cb445f82179f3b208e5fba6df6aa89854dfccb3af0bf74791dba47929';
+const TOPIC_0_TOKEN_SENT_WITH_DATA = '0x875cab82a677ce38c76127a24fa89a67df04116c1a0bf652b61abb89b289f433';
+const TOPIC_0_TOKEN_RECIEVED = '0xa5392cc9825f3ea9fa772e43f2392ca1a9e97db3619eac789383aaaaabb467c4';
```

Comment:
```suggestion
const TOPIC_0_TOKEN_RECEIVED = '0xa5392cc9825f3ea9fa772e43f2392ca1a9e97db3619eac789383aaaaabb467c4';
```
Small typo here and in the rest of the file